### ISSUE TITLE
Add bulk mail option

### ISSUE THREAD

**Issue by kittykat** [puiterwijk/regcfp#222]
Allow a registered user with certain privileges to send mail to

- all registered accounts
- all paper authors (primary and co-presentors)
- all accepted paper authors

When sending mail, a user should be required to:

- have a subject
- have an email body
- select who will receive the email (either drop down or check boxes with above options)
- set a reply to email address (either drop down or freeform)

A copy of the email should be accessible from the GUI when the user is logged in. This is so that late registrations can also access event information.

This year at GUADEC we had three problems: it was too complicated to send bulk mail, only a small number of people could send the mail and the reply to address for paper acceptance/rejection emails was wrong so the papers committee did not receive acks from the presenters.

**Comment by benzea** [comment]
Do we need some sort of moderation feature to ensure that e-mails are proofread by someone other than the writer?

**Comment by kittykat** [comment]
I don't think we need that as part of this feature request, but it could be useful.
